### ISSUE TITLE
[ PowerPoint] :[Comments]: Reply edit field name is  not announced by NVDA

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8282]
### Steps to reproduce:
Turn on NVDA
1.Open PowerPoint and open new/existing presentation containing comments.
2.Press F6 to switch between panes.
3.Select a slide containing comments in thumbnail pane using arrow keys.
4.To Open comments pane navigate to Review Tab-> Show comments button. 
5.Navigate to comments pane using F6.
6.Navigate to Reply edit field using Tab key in comments pane.
 
### Expected behavior:
NVDA should announce the name of reply edit field as “ Reply edit field”

### Actual behavior:
After navigating to Reply edit field NVDA is announcing as 'edit multi line blank'.
NVDA is not announcing the name of the reply edit filed.

### System configuration:
NVDA version: 2018.1.1
App Name: Powerpoint
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by WiproPSAccTest** [comment]
![reply edit_field](https://user-images.githubusercontent.com/22956130/40104789-d9c592fe-590e-11e8-843d-75c847c555e6.png)
![reply_edit](https://user-images.githubusercontent.com/22956130/40104795-dfc79184-590e-11e8-8fea-3862d6a8241c.png)

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
This is not reproducible in Powerpoint 365, NVDA 2024.4 RC reads "slide 1 notes page multi line edit field" when pressing f6 or shift+f6 after adding a comment to the slide.

cc: @CyrilleB79 could you please test under Office 2016 whether this works as well?
